code/game/machinery/doors/poddoor.dm

```diff
@@ -99,6 +102,20 @@
 		balloon_alert(user, "electronics added")
 		deconstruction = BLASTDOOR_FINISHED
 		return ITEM_INTERACT_SUCCESS
+
+	if(deconstruction == BLASTDOOR_FINISHED && istype(tool, /obj/item/assembly/control))
+		if(density)
+			balloon_alert(user, "open the door first!")
+			return ITEM_INTERACT_BLOCKING
+		if(!panel_open)
+			balloon_alert(user, "open the panel first!")
+			return ITEM_INTERACT_BLOCKING
+		var/obj/item/assembly/control/controller_item = tool
+		id = controller_item.id
+		to_chat(user, span_notice("You change the ID to [id]."))
+		balloon_alert(user, "id changed")
```

Comment:
just one or the other, probably the balloon alert

Reply:
I thought about it, but this is intentionally in line with the current multitool id calibration feedback:
https://github.com/tgstation/tgstation/blob/4d33d2f5af218cf381402d7b67970a12ea98964d/code/game/machinery/doors/poddoor.dm#L126-L127
I'll think about it again later